docs/src/content/docs/id/index.mdx

```diff
@@ -39,6 +46,114 @@ import AboutAstro from '~/components/about-astro.astro';
 	</Card>
 </CardGrid>
 
+<TestimonialGrid title="What people are saying">
```

Comment:
The "What people are saying" title can be translated.

Reply:
Ah i miss that one. thank you for the review. will update soon.